### PR TITLE
Feat/training improvements

### PR DESCRIPTION
**Pull request by DonatasTamosauskas**
Added handling for the environment ending cases (`done = True`)

### CODE REVIEW

File: dumbbells/q_functions.py [hunk 1]
```diff
@@ -70,23 +70,30 @@ def __init__(
         self.arch = arch
         self.gamma = gamma
         self.optimizer = (
-            torch.optim.RMSprop(self.arch.parameters()) if optim is None else optim
+            torch.optim.RMSprop(self.arch.parameters(), lr=1e-5)
```
Comment (Z-Nouman) [inline dumbbells/q_functions.py]: We should make learning rate (lr ) a variable so that we can change while fine tuning our model

**Review by Z-Nouman — APPROVED** [review]
Looks good! Only thing I recommend would be to change lr to a variable that's either passed in or a global variable in order to change easily while fine tuning the model later.

File: dumbbells/q_functions.py (hunk 1 above)
Comment (DonatasTamosauskas) [reply dumbbells/q_functions.py]: Nice catch! I didn't realise I left it like so
